cms/io/service.py

```diff
@@ -158,9 +159,11 @@ def _connection_handler(self, sock, address):
 
         """
         try:
-            ipaddr, port = address
-            ipaddr = gevent.socket.gethostbyname(ipaddr)
-            address = Address(ipaddr, port)
+            # In case of IPv6 address is (ip, port, flow info, scope id).
+            ipaddr, port = address[:2]
```

Comment:
In this place, the address is ultimately coming from `accept()`, and should have the numerical form already. So I think we can just create an `Address` directly from that without any address resolution.

Reply:
Right, fixed.